ADVANCED.md

```diff
@@ -154,7 +154,30 @@ function List<T>(props: Props<T>) {
 }
 ```
 
-You can then use them and get nice type safety through type inference:
+The same using fat arrow function style:
+
+```tsx
+interface Props<T> {
+  items: T[];
+  renderItem: (item: T) => React.ReactNode;
+}
+
+const List<T> = <T extends {}>(props: Props<T>) => {
```

Comment:
```suggestion
const List = <T extends {}>(props: Props<T>) => {
```
Variable bindings, or value types, cannot be generic.

Reply:
nice point, thanks for noticing that!